schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -60,4 +66,109 @@ public static void main(String... args) {
     }
     System.exit(status);
   }
+
+  /**
+   * Creates tables defined in the schema file.
+   *
+   * @param configFilePath path to Scalar DB config file.
+   * @param schemaFilePath path to schema file.
+   * @param options specific options for creating tables.
+   * @param createCoordinatorTable create coordinator table or not.\
+   */
+  public static void load(
+      String configFilePath,
+      String schemaFilePath,
+      Map<String, String> options,
+      boolean createCoordinatorTable)
+      throws IOException, SchemaOperatorException {
+    load(
+        configFilePath, schemaFilePath, options, createCoordinatorTable, InputSchemaType.FILE_PATH);
+  }
+
+  /**
+   * Creates tables defined in the schema.
+   *
+   * @param configFilePath path to Scalar DB config file.
+   * @param schema schema definition, can be path to schema file or serialized json string schema.
+   * @param options specific options for creating tables.
+   * @param createCoordinatorTable create coordinator table or not.
+   * @param inputSchemaType type of input schema {@link InputSchemaType}
+   */
+  public static void load(
+      String configFilePath,
+      String schema,
+      Map<String, String> options,
+      boolean createCoordinatorTable,
+      InputSchemaType inputSchemaType)
+      throws IOException, SchemaOperatorException {
```

Comment:
Why it throws two exceptions `IOException` and `SchemaOperatorException`? When we design a method throwing multiple exceptions, we should basically assume they are handled in different ways respectively. Did you assumed something like that?

Reply:
I remove the IOException here and handle it internally. Applied in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/e1abdde1f2580d95c77049cf57ed5b6357c2f978)